index.html

```diff
@@ -336,11 +336,51 @@ <h2 >About me</h2>
             </div>
             
         </section>
-        <section class="form">
+        <footer class="form">
             <div class="txtF">
                 <p class="primary_txt_F"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
             </div>
-        </section>
-
+            <div class="form_f">
+                <form action="https://formspree.io/f/mgeqgykl" method="post">
+                    <input type="text"  id="fullname"  required class="fname" name="fullname" placeholder="Full name" maxlength="30">
+                    
+                   <input type="email" id="fullname" required class="email" name="email" placeholder="Email address">
+                    
+                    <textarea class="txt_area" id="fullname" required placeholder="Enter text here" maxlength="500"></textarea>
+                        <button type="button"  class="button"><p class="btx1">Get in touch</p></button>
```

Comment:
When the submit button is pressed, the form with its data is supposed to be submitted but currently, when the fields are filled and the submit button is pressed, nothing happens.

Luckily, I found out that the problem is because you are using `type="button"` instead of `type="submit"`. Kindly update the type to submit so the form can submit as expected